src/reversediff.jl

```diff
@@ -255,15 +255,23 @@ function logpdf(d::MatrixBeta, X::AbstractArray{<:TrackedMatrix{<:Real}})
     return map(x -> logpdf(d, x), X)
 end
 
-Wishart(df::TrackedReal, S::Matrix{<:Real}) = TuringWishart(df, S)
-Wishart(df::TrackedReal, S::AbstractMatrix{<:Real}) = TuringWishart(df, S)
-Wishart(df::Real, S::TrackedMatrix) = TuringWishart(df, S)
-Wishart(df::TrackedReal, S::TrackedMatrix) = TuringWishart(df, S)
-
-InverseWishart(df::TrackedReal, S::Matrix{<:Real}) = TuringInverseWishart(df, S)
-InverseWishart(df::TrackedReal, S::AbstractMatrix{<:Real}) = TuringInverseWishart(df, S)
-InverseWishart(df::Real, S::TrackedMatrix) = TuringInverseWishart(df, S)
-InverseWishart(df::TrackedReal, S::TrackedMatrix) = TuringInverseWishart(df, S)
+Distributions.Wishart(df::TrackedReal, S::Matrix{<:Real}) = TuringWishart(df, S)
+Distributions.Wishart(df::TrackedReal, S::AbstractMatrix{<:Real}) = TuringWishart(df, S)
+Distributions.Wishart(df::Real, S::AbstractMatrix{<:TrackedReal}) = TuringWishart(df, S)
+Distributions.Wishart(df::TrackedReal, S::AbstractMatrix{<:TrackedReal}) = TuringWishart(df, S)
+Distributions.Wishart(df::Real, S::TrackedMatrix) = TuringWishart(df, S)
+Distributions.Wishart(df::TrackedReal, S::TrackedMatrix) = TuringWishart(df, S)
+Distributions.Wishart(df::Real, S::AbstractPDMat{<:TrackedReal}) = TuringWishart(df, S)
+Distributions.Wishart(df::TrackedReal, S::AbstractPDMat{<:TrackedReal}) = TuringWishart(df, S)
+
+Distributions.InverseWishart(df::TrackedReal, S::Matrix{<:Real}) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::TrackedReal, S::AbstractMatrix{<:Real}) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::Real, S::AbstractMatrix{<:TrackedReal}) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::TrackedReal, S::AbstractMatrix{<:TrackedReal}) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::Real, S::TrackedMatrix) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::TrackedReal, S::TrackedMatrix) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::Real, S::AbstractPDMat{<:TrackedReal}) = TuringInverseWishart(df, S)
+Distributions.InverseWishart(df::TrackedReal, S::AbstractPDMat{<:TrackedReal}) = TuringInverseWishart(df, S)
```

Comment:
Revert these changes? According to the title the PR is supposed to fix only Tracker issues, and I think it's better to keep these changes for the separate ReverseDiff PR (where I just added some comments).